core/cloudflow-operator-actions/src/main/scala/cloudflow/operator/action/EndpointActions.scala

```diff
@@ -50,12 +50,17 @@ object EndpointActions {
 
     val deleteActions = (currentEndpoints -- newEndpoints).flatMap { endpoint ⇒
       Seq(
-        Action.delete[Service](Name.ofService(StreamletDeployment.name(newApp.spec.appId, endpoint.streamlet)), namespace)
+        Action.delete[Service](Name.ofService(StreamletDeployment.name(newApp.spec.appId, endpoint.streamlet)), newApp, namespace)
```

Comment:
I think we can simplify here:
```scala
Action.delete[Service](Name.ofService(StreamletDeployment.name(newApp.spec.appId, endpoint.streamlet)), newApp, namespace)
```

might become something similar to:

```scala
Action.delete[Service](newApp, endpoint.streamlet)
```

Reply:
you might delete in a different namespace.

Reply:
Can add another method, let me see (default to app namespace is probably not bad).

Reply:
done.